docs/documentation/pages/DECKHOUSE-FAQ.md

```diff
@@ -463,6 +463,12 @@ This feature is available in Enterprise Edition only.
    `dhctl mirror` supports digesting of the final set of Deckhouse images with the GOST R 34.11-2012 (Stribog) hash function (the `--gost-digest` parameter).
    The checksum will be logged and written to a file with the `.tar.gostsum` extension next to the tar-archive containing the Deckhouse images.
 
+   Starting from version 1.59, `dhctl mirror` supports splitting the images bundle into chunks of arbitrary size instead of dumping all images into a single tar file.
+   To use this function, pass the desired chunk size in gigabytes with the flag `--images-bundle-chunk-size=N` flag or set the `$MIRROR_IMAGES_BUNDLE_CHUNK_SIZE` environment variable when pulling the bundle.
+   This will create a series of smaller `.chunk` files instead of a single tar bundle.
+   To upload such chunked bundle into your private registry, use `dhctl mirror` as specified below, passing the `--images-bundle-path` as if you were pushing from a single-file bundle.
+   That is, point it to the `.tar` that would not exist in the bundle directory, not any of the chunk files. `dhctl mirror` will then detect the chunked bundle automatically.
```

Comment:
```suggestion
   That is, point it to the `.tar` file as if it were in the bundle directory rather than any of the chunk files. `dhctl mirror` will then detect the chunked bundle automatically.
```